apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableNotToHaveElementsOrAnyExpectationsTest.kt

```diff
@@ -0,0 +1,36 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.fun1
+import ch.tutteli.atrium.specs.withNullableSuffix
+import kotlin.test.Test
+
+class IterableNotToHaveElementsOrAnyExpectationsTest :
+    ch.tutteli.atrium.specs.integration.AbstractIterableNotToHaveElementsOrAnyExpectationsTest(
+        fun1(Expect<Iterable<Double>>::notToHaveElementsOrAny),
+        fun1(Expect<Iterable<Double?>>::notToHaveElementsOrAny).withNullableSuffix()
+    ) {
+
+    @Suppress("AssignedValueIsNeverRead")
```

Comment:
```suggestion
    @Suppress("AssignedValueIsNeverRead", "UNUSED_VALUE")
```

Reply:
That was a mistake on my side, intellij is happy with AssignedValueIsNeverRead so I have put that in the issue description and only realised later on that the compiler isn't. I think I fixed all other places already